tasks/vox.rake

```diff
@@ -0,0 +1,52 @@
+def current_version
+  @current_version ||= File.read(File.expand_path('../CHANGELOG.md', __dir__)).match(/^## \[([^\]]+)\]/) { |match| match[1] }
```

Comment:
I think this is *probably* okay, but we'll have to seed the CHANGELOG with a version first.  I wonder if it makes sense to look up the latest tag instead?